corehq/apps/users/views/__init__.py

```diff
@@ -724,12 +725,22 @@ def __call__(self, request, uuid, **kwargs):
             logout(request)
             return HttpResponseRedirect(request.path)
 
-        invitation = SQLInvitation.objects.filter(uuid=uuid).first()
+        # Recently-sent invitations will use a URL based on UUID
+        try:
+            invitation = SQLInvitation.objects.get(uuid=uuid)
+        except (SQLInvitation.DoesNotExist, ValidationError):
+            invitation = None
+
+        # Older invitations, created before PR#26975, will use a URL based on SQL id
         if not invitation:
```

Comment:
```suggestion
            # Older invitations, created before PR#26975, will use a URL based on SQL id
```

Reply:
You must have been reviewing just as I was merging. I'm going to leave this alone; this code will be deleted in https://github.com/dimagi/commcare-hq/pull/26687